src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -344,7 +310,7 @@ void MulticopterPositionControl::failsafe(const hrt_abstime &now, vehicle_local_
 		} else {
 			// emergency descend with a bit below hover thrust
 			setpoint.vz = NAN;
-			setpoint.acceleration[2] = .3f;
+			setpoint.acceleration[2] = ALTITUDE_THRESHOLD;
```

Comment:
It's not actually an altitude threshold. Not sure if it's better to use a variable that by accident has the same value. How about we create another one with the correct name `EMERGENCY_DESCEND_ACCELERATION` and assign one to the other such that it's readable but the compiler hopefully optimizes the value to be reused?